web-ui/src/components/reviews/TeamReviews.jsx

```diff
@@ -527,7 +527,28 @@ const TeamReviews = ({ onBack, periodId }) => {
     }
   };
 
+  const validateReviewPeriod = period => {
```

Comment:
This function implements review period validation that must hold in order to "REQUEST APPROVAL".